pymeasure/instruments/ophir/ophir_base.py

```
@@ -0,0 +1,596 @@
+
+    def getAllRanges(self):
+        """
+        Get all possible ranges.
+
+        Auto is, if present, -1. The range with the highest numeric value is 0.
+        """
+        # doesn't work with NOVA I
+        ranges = self.values("AR", separator=None, cast=str)
+        index = -1 if ranges[1] == "AUTO" else 0
+        values = {}
+        for range in ranges[1:]:
```

Ruff (0.15.12)

[error] Variable `range` is shadowing a Python builtin

(A001)